### PR TITLE
Migrate to GitHub actions

### PR DESCRIPTION
**Pull request by barmac**

### CODE REVIEW

File: tasks/stages/update-website [hunk 1]
```diff
@@ -26,7 +26,7 @@ if [[ "x$SKIP_COMMIT" = "x" ]]; then
 
   # add all resources
   git add -A
-  git commit -m "chore(project): bump dmn-js to $TRAVIS_TAG"
+  git commit -m "chore: bump dmn-js to $TAG"
```
Comment (pinussilvestrus) [inline tasks/stages/update-website:29]: Maybe we can already use our new conventional commit message:

```sh
deps: bump dmn-js to $TAG
```

Comment (barmac) [reply tasks/stages/update-website:29]: Makes sense. Thanks for the idea.

Comment (barmac) [reply tasks/stages/update-website]: Done.

**Review by pinussilvestrus — APPROVED** [review]
LGTM, let's try it out 👍

**Pull request merged**